libs/deal-ticket/src/components/info-market.tsx

```diff
@@ -18,140 +18,32 @@ import pick from 'lodash/pick';
 import omit from 'lodash/omit';
 import type { MarketInfoQuery, MarketInfoQuery_market } from './__generated__';
 import BigNumber from 'bignumber.js';
-import { gql, useQuery } from '@apollo/client';
+import { useQuery } from '@apollo/client';
 import { totalFees } from '@vegaprotocol/market-list';
-
-const MARKET_INFO_QUERY = gql`
-  query MarketInfoQuery($marketId: ID!) {
-    market(id: $marketId) {
-      id
-      name
-      decimalPlaces
-      positionDecimalPlaces
-      state
-      tradingMode
-      accounts {
-        type
-        asset {
-          id
-        }
-        balance
-      }
-      fees {
-        factors {
-          makerFee
-          infrastructureFee
-          liquidityFee
-        }
-      }
-      priceMonitoringSettings {
-        parameters {
-          triggers {
-            horizonSecs
-            probability
-            auctionExtensionSecs
-          }
-        }
-        updateFrequencySecs
-      }
-      riskFactors {
-        market
-        short
-        long
-      }
-      accounts {
-        type
-        asset {
-          id
-        }
-        balance
-      }
-      data {
-        market {
-          id
-        }
-        markPrice
-        indicativeVolume
-        bestBidVolume
-        bestOfferVolume
-        bestStaticBidVolume
-        bestStaticOfferVolume
-        indicativeVolume
-        openInterest
-      }
-      liquidityMonitoringParameters {
-        triggeringRatio
-        targetStakeParameters {
-          timeWindow
-          scalingFactor
-        }
-      }
-      tradableInstrument {
-        instrument {
-          id
-          name
-          code
-          metadata {
-            tags
-          }
-          product {
-            ... on Future {
-              quoteName
-              settlementAsset {
-                id
-                symbol
-                name
-              }
-              oracleSpecForSettlementPrice {
-                id
-              }
-              oracleSpecForTradingTermination {
-                id
-              }
-              oracleSpecBinding {
-                settlementPriceProperty
-                tradingTerminationProperty
-              }
-            }
-          }
-        }
-        riskModel {
-          ... on LogNormalRiskModel {
-            tau
-            riskAversionParameter
-            params {
-              r
-              sigma
-              mu
-            }
-          }
-          ... on SimpleRiskModel {
-            params {
-              factorLong
-              factorShort
-            }
-          }
-        }
-      }
-      depth {
-        lastTrade {
-          price
-        }
-      }
-    }
-  }
-`;
+import { AccountType, Interval } from '@vegaprotocol/types';
+import { MARKET_INFO_QUERY } from './info-market-query';
 
 export interface InfoProps {
   market: MarketInfoQuery_market;
 }
 
+export const calcCandleVolume = (m: any): string | undefined => {
```

Comment:
chore (blocking): i think the market type should be generated, you don't have to use any for it

Reply:
found it:

```suggestion
export const calcCandleVolume = (m: MarketInfoQuery_market): string | undefined => {
```